src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Menus.vsct

```diff
@@ -214,8 +216,8 @@
   </Commands>
 
   <KeyBindings>
-    <KeyBinding guid="guidManagedProjectSystemOrderCommandSet" id="cmdidMoveUp" editor="SlnExplorerGuid" key1="VK_UP" mod1="Alt" />
-    <KeyBinding guid="guidManagedProjectSystemOrderCommandSet" id="cmdidMoveDown" editor="SlnExplorerGuid" key1="VK_DOWN" mod1="Alt" />    
+    <KeyBinding guid="guidManagedProjectSystemMoveOrderCommandSet" id="cmdidMoveUp" editor="SlnExplorerGuid" key1="VK_UP" mod1="Alt" />
```

Comment:
If these key bindings already exist, why do you need to redefine them?

Reply:
Good point.